docs/introduction/api.md

```diff
@@ -40,20 +71,22 @@ To allow users of your app to contribute without registering individual accounts
 
 ## Help
 
-- Try the FAQ - to answer most of your questions.
-- Didn't get a satisfactory answer? Contact the Team on the Slack Channel. <!---Are we sure the next level of support is from FAQ to slack? If yes, embed links -->
-- Report Bugs on the Open Food Facts Database.
-- Do you have an issue or feature request? You can submit it here [here on GitHub](https://github.com/openfoodfacts/openfoodfacts-server/issues/new).
-- Are you interested in contributing to this project? See your Contribution Guidelines.
+- Try the [FAQ](https://support.openfoodfacts.org/help/en-gb/12-api) - to answer most of your questions.
+- Didn't get a satisfactory answer? Contact the Team on the #api [Slack Channel.](https://slack.openfoodfacts.org/)
+- [Report Bugs](https://github.com/openfoodfacts/openfoodfacts-server/issues/new/choose) on the Open Food Facts Database.
+- Do you have an issue or feature request? You can submit it [here on GitHub](https://github.com/openfoodfacts/openfoodfacts-server/issues/new).
+- Are you interested in contributing to this project? See our [Contribution Guidelines](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/CONTRIBUTING.md).
  <!-- Embed contribution guideline link.-->
 
 ## SDKS
 
-SDKs are available for specific languages to facilitate the usage of the API.
+SDKs are available for specific languages to facilitate the usage of the API. We probably have a wrapper for your favorite programming language. If we do, you can use it and improve it. If we don't, you can help create it.
+
+They will let you consume data and let your users contribute new data.
 
-> **Warning**: The Open Food Facts API reference is the primary documentation source. Endeavor to read it before exploring any SDK.
+> **Warning**: Before exploring any SDK, endeavor to read the **Before You Start** section and the **API Reference Documentation**.
```

Comment:
I think they don't need to read the API Reference, but just to remember that it's the one which is the more accurate.

```suggestion
> **Warning**: Before exploring any SDK, endeavor to read the **Before You Start** section. Also remember, in case of problem, to check the **API Reference Documentation** first to verify if the problem is in SDK implementation or in the API itself.
```